src/content/drafts/custom-rule-disallow-underscores.md

```diff
@@ -0,0 +1,123 @@
+
+---
+layout: post
+title: Create a custom rule to disallow underscores
+teaser: "This blog post explains how to create an ESLint custom rule, which disallows underscores."
+tags:
+  - Custom Rules
+authors:
+  - KICKdesigns
+categories:
+  - Tutorials
+---
+
+
+This blog post describes how to create a custom rule that disallows using an underscore when declaring a variable name. For example, `my-variable` or `myvariable` is be valid, but `my_variable` is disallowed.
+
+To create a custom rule you'll need to:
+
+1. Ensure you complete the [Prerequisites](#prerequisites).
+2. [Create the Plugin](#create-the-plugin).
+3. [Create the Custom Rule](#Create-the-custom-rule).
+
+## Prerequisites
+
+Ensure you complete the following before you proceed:
+
+- Install [Node.js](https://nodejs.org/en/).
+- Install [yarn](https://classic.yarnpkg.com/en/docs/install).
```

Comment:
Yarn should be uppercase throughout (per the project's own usage on yarnpkg.com)